pkg/cmd/project/shared/queries/queries.go

```diff
@@ -335,6 +332,7 @@ func (i DraftIssue) ExportData(_ []string) map[string]interface{} {
 		"body":  i.Body,
 		"type":  "DraftIssue",
 	}
+	// Emulate omitempty.
```

Comment:
Nit: t@samcoe technically, this wasn't to emulate `omitempty` because it never actually was defined as such; however, I reused this method for 2 different cases. That's why I changed one test: it would've practically always had an ID, but the test didn't define it so it came out as `""` instead of absent.

Seemed better to just reuse this function with this condition than define a new function - that way it still an `exportable` in both cases.